src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
also: I do not understand the comment. cuda defines a `__host__ __device__` function for both [powf(float, float)](http://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__SINGLE.html#group__CUDA__MATH__SINGLE) and [pow(double, double)](http://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html#group__CUDA__MATH__DOUBLE)